docs/general/events.md

```diff
@@ -3,13 +3,35 @@ linkTitle: Events
 aliases: [docs/specs/semconv/general/events-general]
 --->
 
-# Semantic Conventions for Event Attributes
+# Semantic Conventions for Events
 
 **Status**: [Experimental][DocumentStatus]
 
-This document describes the attributes of standalone Events that are represented
+This document describes the characteristics of standalone Events that are represented
```

Comment:
Remember that we use "Event" to also means "SpanEvent" - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#add-events

I'm not sure you can just update this section to only be about LogRecord without also touching/describing SpanEvents (which is what this section was initially created to deal with).

Reply:
The intent was not to just describe `LogRecord` as you suggested, but rather to start to flesh out Events as their own first-class signal type. The fact that it they are built on top of or transported via logs should be a separate (implementation) concern. The specific line you commented on changes "attributes" to "characteristic" because Attributes has lots of existing meaning on otel and I didn't want people to think that "attributes" meant "Attributes" while that decision is still being worked on by the events sig.

I don't think we have indicated that Events and SpanEvents are the same thing. @jsuereth are you suggesting that we have? Or maybe we want to?

Reply:
> I'm not sure you can just update this section to only be about LogRecord without also touching/describing SpanEvents (which is what this section was initially created to deal with).

my understanding is that this page only deals with the new experimental (log-based) Events, while Span Events are described here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#add-events

Reply:
My concern is that, reading this, I don't see how it can apply to today's `SpanEvent`.  I like the content you've written and I would otherwise approve this, but I think this will only continue to confuse folks figuring out `SpanEvent` vs. `Event`.

Example confusing language:

```
In addition to a required name, an Event may contain a _payload_ of any type permitted by the
[LogRecord body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body).
In its implementation, the Event _payload_ will constitute the `Body` of the `LogRecord`.
Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe
the event context.
```

How does that interact with this: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#add-events

Reply:
So regarding this concern.  I think we can take the Event/Span-Event confusion into a separate PR or even issue that we track to clean up the language over semconv.  I'd like to make progress here, and I think calling out that log-body has its own fields that are not shared with "Attributes" is important.


Could you open a ticket where we can evaluate and track confusing aspects of the specification going forward regarding "Events (Logs)" vs. "(Span)Events" ?

Reply:
Link to [#695](https://github.com/open-telemetry/semantic-conventions/issues/695).